### PR TITLE
ci: use key locker to sign neuron for windows

### PR DESCRIPTION
**Pull request by Keith-CY**
ref: 
- why use key locker: https://knowledge.digicert.com/solution/digicert-keylocker.html
- how to use key locker in github action: https://docs.digicert.com/en/digicert-keylocker/ci-cd-integrations/script-integrations/github-integration-ksp.html
- Update certificate of Neuron for windows: https://github.com/Magickbase/neuron-public-issues/issues/188

---
Package Neuron for test: https://github.com/nervosnetwork/neuron/actions/runs/6658266030/job/18094724690#step:17:124
image

Package Neuron for Release Draft: https://github.com/nervosnetwork/neuron/actions/runs/6658347469/job/18094990967#step:15:120
image

### CODE REVIEW

File: packages/neuron-wallet/scripts/customSign.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const { execSync } = require('node:child_process')
+
+exports.default = async configuration => {
+  if (!configuration.path) {
+    throw new Error(`Configuration is required`)
```
Comment (Keith-CY) [inline packages/neuron-wallet/scripts/customSign.js:5]:
```suggestion
    throw new Error(`Path of application is not found`)
```

**Comment by yanguoyu** [comment]
How to skip the sign if these `SM_XX` secrets do not exist on the fork repo?

**Comment by Keith-CY** [comment]
> How to skip the sign if these `SM_XX` secrets do not exist on the fork repo?

`Signing application failed` won't block the workflow, e.g. https://github.com/nervosnetwork/neuron/actions/runs/6656736617/job/18089955073#step:13:132

**Review by yanguoyu — APPROVED** [review]

**Comment by Keith-CY** [comment]
> How to skip the sign if these `SM_XX` secrets do not exist on the fork repo?

It does fail if `SM_API_KEY` is not set because authentication is required to download the `smctl` executable, fixed by https://github.com/nervosnetwork/neuron/pull/2913/commits/854c195bdd27780495552e5d14755f232986d79e

**Review by devchenyan — APPROVED** [review]

**Review by devchenyan — APPROVED** [review]

**Pull request merged**